src/features/Send/common/SendContext.tsx

```diff
@@ -14,19 +14,19 @@ export type SendState = {
 
 type TargetActions = {
   amountChanged: (quantity: Quantity) => void
-  amountRemoved: (tokenId: TokenId) => void
+  amountRemoved: (tokenId: string) => void
   receiverChanged: (receiver: string) => void
   addressChanged: (address: Address) => void
 }
 
 type SendActions = {
   yoroiUnsignedTxChanged: (yoroiUnsignedTx: YoroiUnsignedTx | undefined) => void
-  tokenSelectedChanged: (tokenId: TokenId) => void
+  tokenSelectedChanged: (tokenId: string) => void
   resetForm: () => void
   memoChanged: (memo: string) => void
 }
 
-const combinedReducers = (state: SendState, action: SendAction | TargetAction) => {
+export const combinedReducers = (state: SendState, action: SendAction | TargetAction) => {
```

Comment:
why export?

Reply:
oww, it was my first testing approach, :/ deleting